### PR TITLE
Clear release notes - 19.1.0

### PR DESCRIPTION
**Pull request by Itay4**

### CODE REVIEW

File: Integrations/integration-MISP.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ commonfields:
...
-description: Malware Information Sharing Platform and Threat Sharing
+description: Malware Information Sharing Platform and Threat Sharing (This integration is deprecated, use MISP V2 instead)
```
Comment (asafshen) [inline Integrations/integration-MISP.yml]: is this related?

File: Widgets/widget-MttrByType.json [hunk 2]
```diff
@@ -22,6 +22,5 @@
      "keys":["avg|openDuration / 3600"],
      "groupBy" : ["occurred(d)", "type"]
   },
-  "description": "Shows changes in Mean Time to Resolution, over time, while differentiating between incident types.",
-  "releaseNotes": "MTTR now also in timeline widget"
+  "description": "Shows changes in Mean Time to Resolution, over time, while differentiating between incident types."
```
Comment (asafshen) [inline Widgets/widget-MttrByType.json]: is this related

Comment (Itay4) [reply Widgets/widget-MttrByType.json]: Nothing really changed here, just removed release notes

File: Integrations/integration-MISP.yml (hunk 1 above)
Comment (Itay4) [reply Integrations/integration-MISP.yml]: Yes, it's ok
